kolibri/content/constants/presets.py

```diff
@@ -0,0 +1,29 @@
+"""
+This module contains constants representing the ids of FormatPreset.
+"""
+from django.utils.translation import ugettext_lazy as _
+
+# constants for FormatPreset
+HIGH_RES_VIDEO = "high_res_video"
+MID_RES_VIDEO = "mid_res_video"
+LOW_RES_VIDEO = "low_res_video"
+VECTOR_VIDEO = "vertor_video"
+
+EN_SUBTITLE = "en_subtitle"
+ES_SUBTITLE = "es_subtitle"
+ZH_SUBTITLE = "zh_subtitle"
```

Comment:
> Sorry for my slow following, but i am still not quite sure if I understand the issue here. 

No worries, took me quite a while to figure out myself what was causing this! 😜 

> Are you saying some spanish videos have Spanish captions that are translated from English caption, and users who can hear and understand Spanish found the re-translated Spanish caption not correct?

Yep, that's precisely what I'm saying, but it's valid for everybody irrespective of their hearing status. For me it started with having the impression that a lot of subtitles for Spanish videos I was testing last year were poorly translated. It was not only the language inaccuracies that struck me, it was as if they were often out of sync with the video content in question. But if I tested the English video with that same Spanish subtitle, it did make much more sense. That's the rub: KA videos in Spanish (or any other available non-English videos) are **NOT** dubbed (meaning having the same video track with added audio track in a language different from the original one), their video and audio tracks are **re-created** following the Sal's original script. But, no matter how close you try to follow the original script, the translation of transcription/caption of the original English video will be unusable for the newly created video even if they are in the same language.

This probably happened because historically, KA content production flow went something like this:
- Sal makes the original video **A** in English
- at some point KA makes available transcription/caption of that video, let's call it **B0**
- after a while translations of the original captions become available so we have **B1, B2, B3**, etc.
- some partner organization decides to make a version of **A** in their own language so they **re-create A** following Sal's original script. But even if they do that, and the new video topic is identical as **A**, for intents and purposes of captioning this new video is different enough to be **C**, and therefore will not be usable with translated captions **B1, B2 or B3**, even if they are in the same language. 

What we need is a way to **unequivocally** link video **A** with its own captions **B0, B1, B2, B3...**, so none of those captions get (conf)used as correct for videos **C, D, E, F** which are subsequently re-created versions of **A** in other languages for the exactly same topic, made with the same script.

Keep in mind that having the correct captions for any video content is the only way DHH people can actually make use of that video (meaning they are indispensable for them, as opposed to "just" being helpful for the rest of us).